student.rb

```diff
@@ -0,0 +1,16 @@
+require './person'
+class Student < Person
+  def initialize(
+    id:,
+    age:,
+    classroom:, name: 'Unknown',
+    parent_permisson: true
+  )
+    super(id: id, age: age, name: name, parent_permisson: parent_permisson)
+    @classroom = classroom
```

Comment:
- [ ] Super is called correctly however the id can be a random number which you can set in the Person class this will make it not necessary to call here, and the values are passed from the arguments therefore you can write it like this: `super(age, name, parent_permission)`

Reply:
linters does not allowing to do pass parameters like that super(age, name, parent_permission).

Reply:
Fixed